go/interpreter/lfvm/converter_fuzz_test.go

```diff
@@ -0,0 +1,416 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package lfvm
+
+import (
+	"testing"
+
+	"github.com/Fantom-foundation/Tosca/go/ct/gen"
+	"github.com/Fantom-foundation/Tosca/go/tosca/vm"
+
+	"pgregory.net/rand"
+)
+
+func FuzzLfvmConverter(f *testing.F) {
+
+	// Add empty code
+	f.Add([]byte{})
+
+	// Use CT code generator to generate one contract starting with each
+	// opcode
+	rnd := rand.New(1) // deterministic to preserve initial corpus coherence
+	generator := gen.NewCodeGenerator()
+	empty := generator.Clone()
+	for i := 0; i <= 0xFF; i++ {
+		op := vm.OpCode(i)
+		if !vm.IsValid(op) {
+			continue
+		}
+		generator.Restore(empty)
+		generator.SetOperation(0, op)
+		code, err := generator.Generate(gen.Assignment{}, rnd)
+		if err != nil {
+			f.Errorf("Error generating code for opCode %v", op)
+		}
+		f.Add(code.Copy())
+	}
+
+	f.Fuzz(func(t *testing.T, toscaCode []byte) {
+
+		// EIP-170 stablish maximum code size
+		// (see https://eips.ethereum.org/EIPS/eip-170)
+		maxCodeSize := 24_576
```

Comment:
This is actually not the upper boundary. Initiation codes can be twice as large since Shanghai; before that, init codes have been unrestricted. And we need to converting init codes as well.

Reply:
thanks.